src/helpers/routes/routes.js

```diff
@@ -35,6 +35,12 @@ class APIRoutes {
   static veteransMilitaryBranch() {
     return APIRoutes.create('/veterans/get_military_branch');
   }
+  static veteransPhoneNumber() {
```

Comment:
I think we ended up combining this into one route (https://github.com/calblueprint/veterans360-web/blob/master/config/routes.rb#L22) which will return different fields based on whether the veterans are friends (https://github.com/calblueprint/veterans360-web/blob/master/app/controllers/veterans_controller.rb#L100)